docs/workflows/signature.rst

```diff
@@ -7,14 +7,14 @@ how to add signature support.
 
 Setup
 -----
-In order to verify signature validity on uploads you will need to store your trusted keyring on
-your Pulp system and set the ``pulp_ansible`` setting ``ANSIBLE_CERTS_DIR`` to the folder of that
-keyring. By default it is set to ``/etc/pulp/certs/`` if you want an easy place to store your
-keyring.
+In order to verify signature validity on uploads you will need to store your trusted key on the
+repositories ``gpgkey`` attribute.
 
 .. note::
-    You can upload signatures without supplying Pulp your keyring, but ``pulp_ansible`` will not
-    perform validity checks on the uploaded signature.
+   You can upload signatures without supplying Pulp any key, but ``pulp_ansible`` will not
+   perform validity checks on the uploaded signature. You will also have to configure the
+   ``ANSIBLE_SIGNATURE_REQUIRE_VERIFICATION`` setting to ``False``. By default and once a key is
+   provided, all signatures impossible to verify are rejected.
```

Comment:
dies it mean this setting is `True` by default?

Reply:
It currently is, but that's up for debate.

Reply:
I am ok having is as True

If the default is False, then we need to add it to galaxy, but being True require no change on our side.

Reply:
(If we can agree on `False` i would not need to adjust the cli tests right away.)

Reply:
I do not feel comfortable to decide alone. Selecting `True` will change the behaviour for installations using default values. Setting to `False` will accommodate our current CI test suite. But also `True` is heading towards "safe by default".

@gerrod @bmbouter @newswangerd

Reply:
We decided to keep the default true, but i'll try setting it to False in the CI for now to see if the CLI tests start passing again.

Reply:
Whatever you decide is fine with me. We can always change the default on our end if we want.